src/ducks/categories/CategoryChoice.spec.jsx

```diff
@@ -24,66 +24,26 @@ describe('getCategoriesOptions', () => {
   })
 })
 
-describe('CategoryChoice', () => {
-  const setup = ({ categoryId, canSelectParent }) => {
-    const root = mount(
-      <TestI18n>
-        <CategoryChoice
-          canSelectParent={canSelectParent}
-          categoryId={categoryId}
-          onSelect={jest.fn()}
-          onCancel={jest.fn()}
-        />
-      </TestI18n>
-    )
-    return { root }
-  }
-
-  describe('when selecting a normal category', () => {
-    it('should show only one selected item', () => {
-      const { root } = setup({ categoryId: DRESSING_CATEGORY_ID })
-      const selectedRow = findSelectedRow(root)
-      expect(selectedRow.length).toBe(1)
-      expect(selectedRow.text()).toBe('Everyday life')
-      selectedRow.simulate('click')
-
-      const selectedRow2 = findSelectedRow(root)
-      expect(selectedRow2.length).toBe(1)
-      expect(selectedRow2.text()).toBe('Dressing')
-    })
+describe('is selected logic', () => {
+  fit('should mark a category as selected if one of its children is selected', () => {
+    expect(isSelected(dailyLifeOption, dressingOption, 0)).toBe(true)
   })
 
-  describe('when selecting a "others" category', () => {
-    it('should show only one selected item', () => {
-      const { root } = setup({ categoryId: DAILY_LIFE_CATEGORY_ID })
-      const selectedRow = findSelectedRow(root)
-      expect(selectedRow.length).toBe(1)
-      expect(selectedRow.text()).toBe('Everyday life')
-      selectedRow.simulate('click')
-
-      const selectedRow2 = findSelectedRow(root)
-      expect(selectedRow2.length).toBe(1)
-      expect(selectedRow2.text()).toBe('Others : Everyday life')
-    })
+  it('should not mark a category as selected if it is selected', () => {
```

Comment:
The test description is weird. Isn't it « should mark a category as selected if it is selected » ?